conf/conf_example.xml

```diff
@@ -211,12 +211,12 @@
   <aircraft
    name="ardrone2"
    ac_id="201"
-   airframe="airframes/examples/ardrone2.xml"
+   airframe="airframes/TUDelft/airframes/ardrone2_opticflow_stereo_follow_me.xml"
```

Comment:
The conf_example file should not contain so specific config. Please restore the original version here and eventually add this example to the TUDelft conf so it is tested for compilation.